rust/template/ddlog.h

```diff
@@ -1507,4 +1556,24 @@ extern ddlog_cmd* ddlog_delete_val_cmd(table_id table, ddlog_record *rec);
  */
 extern ddlog_cmd* ddlog_delete_key_cmd(table_id table, ddlog_record *rec);
 
+/*
+ * Create a modify-by-key command.
+ *
+ * `table` - input table to delete from.
+ * `key` - key to modify.  The function takes ownership of this record.
+ * `values` - values to modify.  The function takes ownership of this record.
```

Comment:
Not  a good choice of name, imo.  This argument describes a change to apply to the value.  `delta` coule be a better name, but it's already used with a different meaning in this API. Perhaps `patch`?

Reply:
Or modification.